Stripe/PublicHeaders/STPBackendAPIAdapter.h

```diff
@@ -37,7 +35,7 @@ typedef void (^STPCustomerCompletionBlock)(STPCustomer * __nullable customer, NS
  *  @see STPCard
  *  @param completion call this callback when you're done fetching and parsing the above information from your backend. For example, `completion(customer, nil)` (if your call succeeds) or `completion(nil, error)` if an error is returned.
  */
-- (void)retrieveCustomer:(STPCustomerCompletionBlock)completion;
+- (void)retrieveCustomer:(nullable STPCustomerCompletionBlock)completion;
```

Comment:
Hmm, I think it's unlikely for there to be consequences – I can't think of a reason for the developer to ever call this method themselves. I think in the next major version we should make all these `STPBackendAPIAdapter` methods private methods on `CustomerContext`.